cluster-autoscaler/Makefile

```diff
@@ -53,24 +56,24 @@ make-image: make-image-arch-$(GOARCH)
 make-image-arch-%:
 ifdef BASEIMAGE
 	docker build --pull --build-arg BASEIMAGE=${BASEIMAGE} \
-		-t ${REGISTRY}/cluster-autoscaler${PROVIDER}:${TAG}-$* \
+		-t ${IMAGE}-$*:${TAG} \
 		-f Dockerfile.$* .
 else
 	docker build --pull \
-		-t ${REGISTRY}/cluster-autoscaler${PROVIDER}:${TAG}-$* \
+		-t ${IMAGE}-$*:${TAG} \
```

Comment:
```suggestion
		-t ${REGISTRY}/cluster-autoscaler${PROVIDER}-$*:${TAG} \
```

Reply:
Seemed easier to read for me like without using the variable and doesn't seem to be that much more room for mistakes in this case. In any case, it's just a preference.

Reply:
Ack